mlflow/entities/_mlflow_object.py

```diff
@@ -20,3 +21,37 @@ def from_proto(cls, proto):
     @classmethod
     def from_dictionary(cls, the_dict):
         return cls(**the_dict)
+
+    def __repr__(self):
+        return serialize(self)
+
+
+def serialize(obj):
```

Comment:
Just as a note, I think "serialize" can mean converting to formats other than string representation, so I would call this to_string or pretty_print or something.